.changeset/fix_nomissingvarfunction_false_positives_for_container_name.md

```diff
@@ -1,4 +1,5 @@
 ---
+biome_css_analyze: patch
```

Comment:
```suggestion
cli: patch
biome_css_analyze: patch
```

Maybe the contribution guide isn't clear enough. Let me know if it can be improved